web/template/layout/attorney-summary.gohtml

```diff
@@ -41,19 +41,21 @@
               </dd>
             {{ end }}
           </div>
-          
-          <div class="govuk-summary-list__row">
-            <dt class="govuk-summary-list__key ">{{ tr .App "companyEmailAddress" }}</dt>
-            <dd class="govuk-summary-list__value">{{ .TrustCorporation.Email }}</dd>
-            {{ if .CanChange }}
-              <dd class="govuk-summary-list__actions">
-                <a class="govuk-link govuk-link--no-visited-state" href="{{ link .App .Link.TrustCorporation }}#f-email">
-                  {{ trHtml .App "changeCompanyEmailLink" }}
-                </a>
-              </dd>
-            {{ end }}
-          </div>
-          
+
+          {{ if .TrustCorporation.Email }}
```

Comment:
I think we should always show the row, then they can use Change to set one from this page

Reply:
I was following the pattern for attorneys where we hide the row but I agree it would be useful to be able to change this. I'll add in and raise with Beth at next dev review.